packages/stylehacks/src/plugin.js

```diff
@@ -1,59 +1,53 @@
-export default function plugin(targets, nodeTypes, detect) {
-  class Plugin {
-    constructor(result) {
-      this.nodes = [];
-      this.result = result;
-      this.targets = targets;
-      this.nodeTypes = nodeTypes;
-    }
-
-    push(node, metadata) {
-      node._stylehacks = Object.assign({}, metadata, {
-        message: `Bad ${metadata.identifier}: ${metadata.hack}`,
-        browsers: this.targets,
-      });
+export default class Plugin {
+  constructor(result) {
+    this.nodes = [];
+    this.result = result;
+  }
 
-      this.nodes.push(node);
-    }
+  push(node, metadata) {
+    node._stylehacks = Object.assign({}, metadata, {
+      message: `Bad ${metadata.identifier}: ${metadata.hack}`,
+      browsers: this.targets,
+    });
 
-    any(node) {
-      if (this.nodeTypes.includes(node.type)) {
-        detect.apply(this, arguments);
+    this.nodes.push(node);
+  }
 
-        return !!node._stylehacks;
-      }
+  any(node) {
+    if (this.nodeTypes.includes(node.type)) {
+      this.detect(node);
 
-      return false;
+      return !!node._stylehacks;
     }
 
-    detectAndResolve(...args) {
```

Comment:
This function only work with a single `Node`. I suspect rest parameters were used to obtain an array to pass to `apply`.